ECart/Cart.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+
+namespace ECart
+{
+    class Cart
+    {
+        List<CartItem> ItemList;
+        double CartTotal = 0, TotalDiscount = 0, DiscountedTotal = 0;
+        double DiscountPercentage;
+        public Cart()
+        {
+            ItemList = new List<CartItem>();
+        }
+        public void DiscountOnTotal(double DiscountPercentage = 25)
```

Comment:
local variables should be in camel case. discountPercentage.

Reply:
Resolved